sbin/upload-artifacts

```diff
@@ -96,7 +89,7 @@ s3_upload_file() {
 	local s3_dir="$2"
 	[[ $s3_dir != */ ]] && s3_dir="${s3_dir}/"
 	
-	$OP aws s3 cp $file $s3_dir --acl public-read --no-progress
+	$OP aws --debug s3 cp $file $s3_dir --acl public-read --no-progress
```

Comment:
is this a leftover or for verbosity?

Reply:
I left it on purpose, I think it can help our debugability (without it we get almost no information upon failures)